bridge_ui/src/utils/consts.ts

```diff
@@ -251,7 +251,7 @@ export const getDefaultNativeCurrencySymbol = (chainId: ChainId) =>
     : chainId === CHAIN_ID_BSC
     ? "BNB"
     : chainId === CHAIN_ID_TERRA
-    ? "LUNA"
+    ? "LUNAC"
```

Comment:
is it LUNC instead of LUNAC?
https://www.coingecko.com/en/coins/terra-luna-classic